### ISSUE TITLE
webpage owner should have control

### ISSUE THREAD

**Issue by rocky-III** [fiatjaf/nocomment#19]
right now the script generates automatically a nostr identity/keys  over which the webpage owner has no control. It would be more interesting if the a webpage owner could create this identity/keys by himself and build the widget with it.

**Comment by fiatjaf** [comment]
I agree, but this is not really an issue. This widget is a proof-of-concept waiting for someone else more capable and with more time to implement something better.

**Comment by rocky-III** [comment]
well take it as a feature request... but actually I think it is a big issue.
I liked the concept first and implemented the widget as a try on my HP - but now i think I may better deactivate quid soon again while i have absolutely  no control over what is posted with reference to my HP address - Is there a way to delete the automatic created account on snort.social  with my HP address ?
